src/components/Editor/CalendarPickerHeader.vue

```diff
@@ -82,6 +85,11 @@ export default {
 			return this.isReadOnly || this.calendars.length < 2
 		},
 	},
+	mounted() {
+		// Taken from https://pictogrammers.com/library/mdi/icon/menu-down/
+		const menuDownIconUrl = 'url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCI+PHBhdGggZD0iTTcsMTBMMTIsMTVMMTcsMTBIN1oiIC8+PC9zdmc+)'
```

Comment:
do we need any attribution or point to a license?

Reply:
Good point. Google doesn't require it[^1] but I added it anyway.

[^1]: https://developers.google.com/fonts/docs/material_icons#licensing